frameworks/elastic/universe-kibana/config.json

```diff
@@ -14,6 +14,12 @@
           "description": "The user that runs the Kibana services and owns the Mesos sandbox.",
           "type": "string",
           "default": "nobody"
+        },
```

Comment:
Oops, that wasn't my question...somehow the comment got attached to the wrong line. I was asking about the relevance of log4j2 configuration w/r/t a Marathon app.